economy/static/economy/css/economy_home.css

```diff
@@ -0,0 +1,89 @@
+.card {
+  display: flex;
+  align-items: center;
+
+  color: #697a92;
+
+  border: 2px solid #e3e7ef;
+  border-radius: 0.75px;
+  box-shadow: 0 3px 6px rgba(0, 0, 0, 0.16);
+
+  padding: 1rem;
+
+  margin: 0 0 1rem 0;
+}
+
+.card > * {
+  padding: 1rem;
+}
+
+.justify-center {
+  justify-content: center;
+}
+
+#economy-home-wrapper {
+  display: flex;
+  flex-direction: column;
+
+  align-items: center;
+}
+
+#economy-home-grid {
+  display: grid;
+  max-width: 80%;
+  grid-template-columns: 1fr 1fr;
+  gap: 10px;
+  grid-template-areas:
+    "tl tr"
+    "bl br";
+}
+
+#economy-home-grid > #spent {
+  grid-area: tr;
+  padding: 1rem;
+}
+
+#economy-home-grid > #balance {
+  grid-area: tl;
+  padding: 1rem;
+}
+
+#economy-home-grid > #latest-purchases {
+  grid-area: bl;
+  padding: 1rem;
+}
+
+#economy-home-grid > #latest-deposits {
+  grid-area: br;
+  padding: 1rem;
+}
+
+.date {
+  border-right: 2px solid #e3e7ef;
+}
+
+.icon {
+  font-size: 200%;
+}
+
+.d-flex-row-between {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+}
+
+input,
+select,
+textarea {
+  width: 100%;
```

Comment:
dont use universal tag selectors, could probably end up with some namspace conflict down the line which will end up being hard to debug